src/components/Modal/Modal.jsx

```diff
@@ -440,6 +446,8 @@ Modal.propTypes = {
   description: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
   /** Used with `path` and React Router. Renders if path matches _exactly_ */
   exact: PropTypes.bool,
+  /** If you don't want the focus to be moved to the Modal when it enters */
+  focusModalOnShow: PropTypes.bool,
```

Comment:
Looks like this prop doesn't have a default value, so I'm guessing it's defaulting to `undefined`, right? Should this default to `true` in order to maintain backward compatibility?

Reply:
Yes! I forgot to add it thanks 👍